src/routes/dorm/reservation/dorm-a-discussion-room.tsx

```
@@ -0,0 +1,8 @@
+import DiscussionRoomReservation from '@/pages/dorm/reservation/discussion-room'
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)